### PR TITLE
Adding Traffic "network" property

### PR DESCRIPTION
**Pull request by conor-ettinoffe-here**
- Adding "network" as an optional property to Traffic Flow and Incidents layers as per the question raised here: https://github.com/tilezen/vector-datasource/issues/1935

- Also cleaning up a bit by referencing headers within this file rather than full URLs

### CODE REVIEW

File: docs/layers.md [hunk 1]
```diff
@@ -2290,12 +2290,13 @@ If the roads layer features include `linear_ref_id`, then traffic flow layer fea
 
 However, if the roads layer does not include `linear_ref_id`, then it should be delivered with geometry and the following properties to enable sizing and layering of the traffic overlay with respect to the roads network:
 
-* `road_kind`: [kind of the road](https://github.com/tilezen/vector-datasource/blob/master/docs/layers.md#road-properties-common)
-* `road_kind_detail`: [kind_detail of the road](https://github.com/tilezen/vector-datasource/blob/master/docs/layers.md#road-properties-common)
+* `road_kind`: [kind of the road](#road-properties-common)
```
Comment (nvkelso) [inline docs/layers.md:2293]: Thanks!

File: docs/layers.md [hunk 2]
```diff
@@ -2290,12 +2290,13 @@ If the roads layer features include `linear_ref_id`, then traffic flow layer fea
...
+* `road_kind_detail`: [kind_detail of the road](#road-properties-common)
 * `is_bridge`: set to `true` when the linear is a bridge
 * `is_tunnel`: set to `true` when the linear is a tunnel
 * `is_link`: set to true when the linear is a slip-road
 * `is_hov_lane`: set to `true` when the linear is an HOV (High-Occupancy Vechicle) express lane
+* `network`: 2-character country code and optional region (eg. DE:Berlin) of the traffic information (See: [Roads Layer Network Values](#roads-layer-network-values))
```
Comment (nvkelso) [inline docs/layers.md:2299]: The example should be in ticks like:

> e.g. `DE:Berlin`

But I don't think `DE:Berlin` is a valid network code. It should match one of the existing network values for Germany that are listed in the linked section. Namely, one of:

- `DE:BAB`
- `DE:BS`
- `DE:Hamburg:Ring`
- `DE:KS`
- `DE:LS`
- `DE:STS`
- `DE`

File: docs/layers.md [hunk 3]
```diff
@@ -2344,14 +2345,14 @@ If the roads layer features include `linear_ref_id`, then traffic incidents laye
...
-* `road_kind_detail`: [kind_detail of the road](https://github.com/tilezen/vector-datasource/blob/master/docs/layers.md#road-properties-common)
+* `road_kind`: [kind of the road](#road-properties-common)
+* `road_kind_detail`: [kind_detail of the road](#road-properties-common)
 * `is_bridge`: set to `true` when the linear is a bridge
 * `is_tunnel`: set to `true` when the linear is a tunnel
 * `is_link`: set to true when the linear is a slip-road
 * `drives_on_left`: set to `true` when the country drives on the left (e.g. in the United Kingdom)
 * `is_hov_lane`: set to `true` when the linear is, or the incident affects, a HOV (High-Occupancy Vechicle) express lane
-
+* `network`: 2-character country code and optional region (eg. DE:Berlin) of the traffic information (See: [Roads Layer Network Values](#roads-layer-network-values))
```
Comment (nvkelso) [inline docs/layers.md:2355]: Same comment as above.

File: docs/layers.md (hunk 1 above)
Comment (conor-ettinoffe-here) [reply docs/layers.md:2293]: :+1:

File: docs/layers.md (hunk 2 above)
Comment (conor-ettinoffe-here) [reply docs/layers.md]: Good catch!

File: docs/layers.md (hunk 3 above)
Comment (conor-ettinoffe-here) [reply docs/layers.md]: Fixed

**Pull request merged**